src/core/service.c

```diff
@@ -1263,10 +1262,6 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        r = unit_full_printf_strv(UNIT(s), c->argv, &argv);
-        if (r < 0)
```

Comment:
Grmbl.. THis kinds was always broken... I mean this already results in confusion:

```
[Unit]
Description=A unit whose cgroup path is %c
[Service]
ExecStart=/bin/echo %c
```

with old systemd the description string would show the wrong path, but the exec line the wrong one...

with my patch both now show the wrong path... I guess making already broken things completely broken isn't considered a "bugfix" is it? ;-)

Meh, "%c" should never have been added...